### PR TITLE
feat: Support to Prompt metadata API

### PR DESCRIPTION
**Pull request by froucher**
https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_prompt.htm

### CODE REVIEW

**Comment by froucher** [comment]
PR #241

File: test-functional/data/simple-metadata/prompt-removed/v0/package.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Package xmlns="http://soap.sforce.com/2006/04/metadata">
+    <types>
```
Comment (amtrack) [inline test-functional/data/simple-metadata/prompt-removed/v0/package.xml:3]: @froucher Please remove these lines with the wildcard for Promt (from all package.xml files).

Comment (amtrack) [reply test-functional/data/simple-metadata/prompt-removed/v0/package.xml:3]: @froucher there are two more wildcards:

- `test-functional/data/simple-metadata/prompt-updated/v0/package.xml`
- `test-functional/data/simple-metadata/prompt-updated/v1/package.xml`

**Comment by froucher** [comment]
Done. I want to add support to LeadConvertSettings too, do you want a new pull request? or you'd prefer to use this PR?

**Comment by amtrack** [comment]
> Done. I want to add support to LeadConvertSettings too, do you want a new pull request? or you'd prefer to use this PR?

In this case a separate PR please.

**Comment by froucher** [comment]
Thank you!

**Pull request merged**